app/services/user_tower_service.py

```
@@ -0,0 +1,40 @@
+            catetory_text = ", ".join(profile.interest_categories)
+            text_parts.append(f"{catetory_text} 제품에 관심이 많습니다.")
+
+        text_parts.append(f"가격대는 {profile.min_price}원에서 {profile.max_price}원 사이를 선호합니다.")
+
+        if profile.additional_info:
+            text_parts.append(profile.additional_info.strip())
+
+        return " ".join(text_parts)
+    
+
+    def generate_user_embedding(self, profile: BeautyProfile) -> np.ndarray:
```

Flake8 (7.2.0)

[error] too many blank lines (2)

(E303)